components/library/tntnet/tntnet-server.p5m

```diff
@@ -33,5 +33,5 @@ file path=usr/share/man/man7/tntnet.xml.7
 
 file path=usr/share/tntnet/tntnet-conf2xml.pl
 
-file path=usr/etc/init.d/tntnet
-file path=usr/etc/tntnet/tntnet.xml
+file path=etc/init.d/tntnet
```

Comment:
SMF? or is there a "service" pkg(5) action? :-)
At least today, SMF is not likely; if the rest is acceptable I'd prefer to leave it for another PR.